testsuite/MDAnalysisTests/analysis/test_leaflet.py

```diff
@@ -42,3 +44,11 @@ def test_leaflet_finder(self):
         assert_equal(bottom_heads.indices, np.arange(2521,4670,12), err_msg="Found wrong leaflet lipids")
 
 
+    def test_string_vs_atomgroup_proper(self):
+        from MDAnalysis.analysis.leaflet import LeafletFinder
```

Comment:
Are we doing the in-method imports to work around some potentially missing modules? Otherwise it should be at the top of the file.